projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.holo.craft.materials.stat.tool_level.tooltip": "使用这种材料合成的工具的基础等级",
+    "tetra.holo.craft.materials.stat.tool_efficiency": "工具效率",
+    "tetra.holo.craft.materials.stat.tool_efficiency.tooltip": "使用这种材料合成的工具的基础效率",
+    "tetra.holo.craft.materials.stat.durability": "耐久",
+    "tetra.holo.craft.materials.stat.durability.tooltip": "使用这种材料合成的工具的基础耐久",
+    "tetra.holo.craft.materials.stat.integrity": "完整度",
+    "tetra.holo.craft.materials.stat.integrity.tooltip": "使用这种材料合成的组件的基础完整度增益或消耗（增益 | 消耗）",
+    "tetra.holo.craft.materials.stat.magic_capacity": "魔力",
+    "tetra.holo.craft.materials.stat.magic_capacity.tooltip": "使用这种材料合成的主要组件的魔力容量",
+    "tetra.holo.craft.materials.stat_modifier": "修正",
+    "tetra.holo.craft.materials.stat_modifier.tooltip": "使用这种材料合成的主要组件会提供§e%s§r修正：",
+    "tetra.holo.craft.materials.stat_effect": "效果",
+    "tetra.holo.craft.materials.stat_effect.tooltip": "使用这种材料合成的主要组件会提供§e%s§r效果：",
+    "tetra.workbench.slot_detail.details_tab": "详情",
+    "tetra.workbench.slot_detail.craft_tab": "合成",
+    "tetra.workbench.slot_detail.tweak_tab": "调整",
+    "tetra.workbench.slot_detail.tweak_apply": "应用",
+    "tetra.workbench.module_detail.empty": "空槽位",
+    "tetra.workbench.module_detail.not_tweakable": "调整不可用",
+    "tetra.workbench.schematic_list.empty": "所选插槽没有可用的升级。",
+    "tetra.workbench.schematic_detail.craft": "合成",
+    "tetra.workbench.schematic_detail.back": "返回",
+    "tetra.workbench.schematic_detail.experience": "此操作需要 %d 级经验",
+    "tetra.workbench.schematic_list.temporary": "你的背包中有可用物品",
+    "tetra.repair_action.label": "修理",
+    "tetra.tool.hammer": "锤",
+    "tetra.tool.hammer.requirement": "此操作需要 %d 级的锤",
+    "tetra.tool.axe": "斧",
+    "tetra.tool.axe.requirement": "此操作需要 %d 级的斧",
+    "tetra.tool.pickaxe": "镐",
+    "tetra.tool.pickaxe.requirement": "此操作需要 %d 级的镐",
+    "tetra.tool.shovel": "锹",
+    "tetra.tool.shovel.requirement": "此操作需要 %d 级的锹",
+    "tetra.tool.cut": "切割",
+    "tetra.tool.cut.requirement": "这一步需要一个 %d 级的切割工具。\n§7大部分的剑可以被用于切割。",
+    "tetra.tool.pry": "撬",
+    "tetra.tool.pry.requirement": "这一步需要一个 %d 级的撬具。\n§7羊角头可以被用于撬。",
+    "tetra.tool.available": "你有一 个%d 级的",
+    "tetra.stats.efficiency_suffix": "效率",
+    "tetra.stats.strength_suffix": "力量",
+    "tetra.stats.tier_suffix": "等级",
+    "tetra.stats.level_suffix": "等级",
+    "tetra.stats.reach": "使用范围",
+    "tetra.stats.reach.tooltip": "可以触及最远 §e%s§r 格的方块和实体",
+    "tetra.stats.reach_normalized": "使用范围",
+    "tetra.stats.reach_normalized.tooltip": "增加这个物品的使用范围 §e%s§r 个方块",
+    "tetra.stats.durability": "耐久",
+    "tetra.stats.durability.tooltip": "这个物品有 §e%s§r 点耐久",
+    "tetra.stats.speed": "速度",
+    "tetra.stats.speed.tooltip": "影响物品可以被挥动得多快\n \n§e%s §7秒的挥动间隔\n§e%s§7 倍的采集效率",
+    "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击会造成更少的伤害。横扫或急速打击之类的效果只有在攻击指示条完全充满时才会被触发。§7速度值相当于采集效率的倍率。取决于大于或小于 1,它会增加或减少方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 * 0.5 + 0.5§r§7",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
    "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击会造成更少的伤害。横扫或急速打击之类的效果只有在攻击指示条完全充满时才会被触发。§7速度值相当于采集效率的倍率。取决于大于或小于 1，它会增加或减少方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 * 0.5 + 0.5§r§7",
```

---

除此之外，「之类的」一词明显不够书面。